### PR TITLE
docs: updated the supported execution mode of limit clause

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)
there are some Chinese syntax errors like line 3
the execution modes which supported LIMIT are outdated
* **What is the new behavior (if this is a feature change)?**
updated the supported execution mode of LIMIT clause
fixed syntax srrors

### CODE REVIEW

File: docs/zh/reference/sql/dql/LIMIT_CLAUSE.md [hunk 1]
```diff
@@ -17,16 +17,15 @@ SELECT ... LIMIT ...
...
+| LIMIT Clause   | 单机版和集群版的所有执行模式均支持 | Limit 子句用于限制返回的结果条数。目前Limit仅支持接受一个参数，表示返回数据的最大行数。 |
 
 ## Example
 
 ### SELECT with LIMIT
 
 ```SQL
--- desc: SELECT Limit
-  SELECT t1.COL1 c1 FROM t1 limit 10;
+>SELECT t1.COL1 c1 FROM t1 limit 10;
```
Comment (vagetablechicken) [inline docs/zh/reference/sql/dql/LIMIT_CLAUSE.md]: '>' is needless?

Comment (michelle-qinqin) [reply docs/zh/reference/sql/dql/LIMIT_CLAUSE.md]: Just want to correspond with the format in quickstart.
[快速上手/OpenMLDB快速上手](https://openmldb.ai/docs/zh/main/quickstart/openmldb_quickstart.html)

Comment (michelle-qinqin) [reply docs/zh/reference/sql/dql/LIMIT_CLAUSE.md]: I will commit a new version later

Comment (lumianph) [reply docs/zh/reference/sql/dql/LIMIT_CLAUSE.md]: i agree > is unnecessary for those docs that are clearly known to be SQLs

File: docs/zh/reference/sql/dql/LIMIT_CLAUSE.md [hunk 2]
```diff
@@ -17,16 +17,15 @@ SELECT ... LIMIT ...
 
 ## 边界说明
 
-| SELECT语句元素 | 边界                 | 说明                                                         |
-| :------------- | -------------------- | :----------------------------------------------------------- |
-| LIMIT Clause   | 不支持Online Serving | Limit 子句用于限制结果条数。OpenMLDB 目前仅支持Limit 接受一个参数，表示返回数据的最大行数； |
+| SELECT语句元素 | 边界 | 说明                                                         |
+| :------------- |--| :----------------------------------------------------------- |
+| LIMIT Clause   | 单机版和集群版的所有执行模式均支持 | Limit 子句用于限制返回的结果条数。目前Limit仅支持接受一个参数，表示返回数据的最大行数。 |
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/LIMIT_CLAUSE.md:22]: "单机版和集群版的所有执行模式均支持"
is this confirmed?

Comment (michelle-qinqin) [reply docs/zh/reference/sql/dql/LIMIT_CLAUSE.md:22]: it is supported by standalone version, offline mode, online request mode and online preview mode according to the newest table. Should I change the description as "单机版、集群版的离线模式、在线预览模式、在线请求模式均支持"？

Comment (lumianph) [reply docs/zh/reference/sql/dql/LIMIT_CLAUSE.md:22]: > it is supported by standalone version, offline mode, online request mode and online preview mode according to the newest table. Should I change the description as "单机版、集群版的离线模式、在线预览模式、在线请求模式均支持"？

i think no need, the current description is fine, just to double check

**Review by lumianph — APPROVED** [review]

**Pull request merged**